nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -221,6 +221,10 @@ in
           NOTE: this will override default listening on all local addresses and port 22.
           NOTE: setting this option won't automatically enable given ports
           in firewall configuration.
+          NOTE: If the IP address is not available at boot time, the following has
```

Comment:
Can you refer to the issue you mentioned in the documentation? Thank you!

Reply:
Added a link, if the formatting is acceptable?